samples/dotnet/KernelHttpServer/KernelHttpServer.csproj

```diff
@@ -26,7 +26,9 @@
   <ItemGroup>
```

Comment:
The folder name should be `kernel-http-server` to follow our pattern I believe.

Reply:
we can rename both in a separate PR